lib/langchain/llm/response/ollama_response.rb

```diff
@@ -44,5 +44,9 @@ def prompt_tokens
     def completion_tokens
       raw_response.dig("eval_count")
     end
+
+    def summarization
```

Comment:
I'm not sure we need this method. Since it's basically just a completion based on the prompt instructions -- I think it's okay to just use `.completion`.